tests/AnonymousUserTests/AnonymousUserTests.csproj

```diff
@@ -0,0 +1,25 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>net5.0</TargetFramework>

```

Comment:
You can also use net6.0 if you upp the appveyor config file to use the Visual Studio 2022 image. I'd suggest to run the tests for all LTS runtimes (`netcoreapp3.1`, `net6.0`) and maybe even `net5.0`.

https://www.appveyor.com/docs/windows-images-software/
https://dotnet.microsoft.com/en-us/download/dotnet

Reply:
Yeah, I have added all 3 runtimes. Although I don't think this will make a big difference for this lib

Reply:
I agree. Ideally we should change the lib to target different versions of ASP.NET Core depending on the target framework but we can supersede for now.